tasks/service-runner/config/query-cmr-sidecar.yaml

```diff
@@ -76,6 +76,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: MY_POD_UID
```

Comment:
It looks like the sidecar and service-template files are only used locally. We'll need a change to the ci-cd repo as well to add `MY_POD_UID`.